### PR TITLE
[Rule Tuning] Windows BBR Rule Tuning - 3

### PR DESCRIPTION
**Pull request by w0rk3r**
## Issues

Related to #3551 

## Summary

Tighten up indexes and exclude a few noisy patterns.

### CODE REVIEW

File: rules_building_block/persistence_startup_folder_lnk.toml [hunk 1]
```diff
@@ -33,8 +33,10 @@ file where host.os.type == "windows" and event.type != "deletion" and file.exten
     "C:\\ProgramData\\Microsoft\\Windows\\Start Menu\\Programs\\StartUp\\*"
   ) and
   not (
-    (process.name : "ONENOTE.EXE" and process.code_signature.status: "trusted" and file.name : "Send to OneNote.lnk") or
-    (process.name: "OktaVerifySetup.exe" and process.code_signature.status: "trusted" and file.name : "Okta Verify.lnk")
+    (process.name : "ONENOTE.EXE" and process.code_signature.status: "trusted" and file.name : "*OneNote.lnk") or
```
Comment (Aegrah) [inline rules_building_block/persistence_startup_folder_lnk.toml:36]: Don't we have `file.path`s that are predictable here to not rely on starting wildcards?

**Review by Aegrah — APPROVED** [review]

File: rules_building_block/persistence_startup_folder_lnk.toml (hunk 1 above)
Comment (Samirbous) [reply rules_building_block/persistence_startup_folder_lnk.toml:36]: don't think EG has code signature info for file event ?

Comment (w0rk3r) [reply rules_building_block/persistence_startup_folder_lnk.toml:36]: > Don't we have file.paths that are predictable here to not rely on starting wildcards?

The file.name here is dependent on the system language, so we have values like: `Invia a OneNote.lnk`, `Send til OneNote.lnk`, `Envoyer à OneNote.lnk`, `Send to OneNote.lnk`, etc

Comment (w0rk3r) [reply rules_building_block/persistence_startup_folder_lnk.toml:36]: > don't think EG has code signature info for file event ?

Hmmm, that is right, do you think we should remove the process signature check or endgame compat?

File: rules_building_block/execution_settingcontent_ms_file_creation.toml [hunk 2]
```diff
@@ -32,7 +32,10 @@ type = "eql"
 query = '''
 file where host.os.type == "windows" and event.type == "creation" and
   file.extension : "settingcontent-ms" and
-  not file.path : "?:\\Users\\*\\AppData\\Local\\Packages\\windows.immersivecontrolpanel_*\\LocalState\\Indexed\\Settings\\*"
+  not file.path : (
+    "*\\AppData\\Local\\Packages\\windows.immersivecontrolpanel_*\\LocalState\\Indexed\\Settings\\*",
```
[the remark this answers is not in the document]
Comment (Samirbous) [reply rules_building_block/execution_settingcontent_ms_file_creation.toml:36]: that's weird indeed, maybe just add `?:\\*\\...`

**Review by Samirbous — APPROVED** [review]

File: rules_building_block/execution_settingcontent_ms_file_creation.toml (hunk 2 above)
[the remark this answers is not in the document]
Comment (w0rk3r) [reply rules_building_block/execution_settingcontent_ms_file_creation.toml:36]: Will do it

File: rules_building_block/persistence_startup_folder_lnk.toml (hunk 1 above)
Comment (Samirbous) [reply rules_building_block/persistence_startup_folder_lnk.toml:36]: ++ EG compat is better as OneNote process path may require extra wildcards for users homeprofile

**Pull request merged**